iphone/Classes/TiUISearchBarProxy.m

```diff
@@ -184,9 +184,52 @@ - (id)barColor
 
 - (NSNumber *)style
 {
-  return NUMINT([[self searchBar] searchBarStyle]);
+  return @([[self searchBar] searchBarStyle]);
 }
 
+#if IS_SDK_IOS_13
+- (void)insertTokenAtIndex:(id)params
+{
+  ENSURE_ARG_COUNT(params, 2);
+
+  if (![TiUtils isIOSVersionOrGreater:@"13.0"]) {
+    return;
+  }
+
+  NSDictionary<NSString *, NSString *> *token = params[0];
+  int index = [TiUtils intValue:params[1]];
+
+  UISearchToken *searchToken = [UISearchToken tokenWithIcon:[TiUtils toImage:token[@"image"] proxy:self]
+                                                       text:[TiUtils stringValue:@"text" properties:token]];
+
+  if (token[@"identifier"] == nil) {
+    NSLog(@"[WARN] Missing search token identifier! Using a generated UUID …");
+  }
+
+  searchToken.representedObject = [TiUtils stringValue:@"identifier" properties:token def:[TiUtils createUUID]];
```

Comment:
Property 'representedObject' is nullable. I think there is no need to assign any default value if it is not passed.

Reply:
It's required to be able to identify them when updating / deleting them.